### ISSUE TITLE
Clarify effect on module loader cache and module reuse

### ISSUE THREAD

**Issue by mikesamuel** [mikesamuel/dynamic-import-host-adjustment#3]
Host internal caches key on strings to prevent repeated fetching and loading of semantically-equivalent module references.  This proposal should not affect or complicate that.

@bmeck, you raised during plenary that implementations often have corner cases around garbage collection with cross-realm GC.  Do I understand correctly that this is a concern re module-reference-string-wrapping objects as keys into per-realm caches related to `import(moduleReferenceFromAnotherRealm)`?  Or is that a separable issue?

**Comment by bmeck** [comment]
This is true, I have lots of docs on this from our node loader work, but would prefer if we could sync up and try and disseminate what the pieces that truly affect the intent of the proposal sometime.

**Comment by mikesamuel** [comment]
@bmeck That sounds lovely.  Are you perchance in Montreal next week?

**Comment by bmeck** [comment]
@mikesamuel I'm not likely to be anywhere until my baby is a bit older (next Sept. maybe). A recorded video call would be ideal since we could publish it for records if we choose to do so.

**Comment by mikesamuel** [comment]
Understood.  I've been there :)

I'll send you a doodle.  I'll ping MarkM and KevinG in case they're interested too.
